src/index.html

```diff
@@ -24,8 +24,91 @@
         <link rel="preconnect" href="https://connect.facebook.net" crossorigin="anonymous">
         <link rel="prefetch" href="<%= process.env.API_ORIGIN %>/apps/cms/api/books/?format=json">
         <link rel="prefetch" href="<%= process.env.API_ORIGIN %>/apps/cms/api/footer/?format=json">
-        <link rel="stylesheet" href="https://ram.openstax.org/osano/osano.css">
-        <script src="https://cmp.osano.com/AzZqbXTbzhHsU3cv1/68d8e8ae-4024-4489-a000-72766ad284a6/osano.js"></script>
+        <script>
+          // This code is shared with the accounts-model
+          window._OX_USER_PROMISE ||= fetch('/accounts/api/user?always_200=true', {credentials: 'include'}).then(
+              (response) => response.json().catch(
+                  (err) => {
+                      console.warn('No JSON in Accounts response');
+                      return {err};
+                  }
+              ),
+              (err) => {
+                  console.warn('"Error fetching user info"');
+                  return {err};
```

Comment:
doesn't this allow the `{err}` object to proceed to the success handler

Reply:
Moved around as we discussed